apps/backend/src/handlers/get.ts

```diff
@@ -1,27 +0,0 @@
-import { APIGatewayEvent } from 'aws-lambda';
-import { createTableIfNotExists } from '../db/createTable.js';
-
-// Create clients and set shared const values outside of the handler.
-/**
- * A simple example includes a HTTP get method that returns "Hello world!".
- */
-export const getHandler = async (event: APIGatewayEvent) => {
-  if (event.httpMethod !== 'GET') {
-    throw new Error(
-      `getHandler only accept GET method, you tried: ${event.httpMethod}`
-    );
-  }
-  // All log statements are written to CloudWatch
-  console.info('received:', event);
-  await createTableIfNotExists();
-
-  const response = {
```

Comment:
actually we might want to keep this as a "Hello world" endpoint so we can easily test if our backend is up and running